src/ui/pages/BridgeForm/SuccessState/SuccessState.tsx

```diff
@@ -81,9 +89,7 @@ export function SuccessState({
         pendingTitle="Transferring"
         failedTitle="Transfer failed"
         dropppedTitle="Transfer cancelled"
-        explorerUrl={
-          hash ? networks.getExplorerTxUrlByName(spendChain, hash) : undefined
-        }
+        explorerUrl={explorerUrl ?? explorerFallbackUrl}
```

Comment:
as we discussed It makes more sense to display links like [this](https://relay.link/transactions?txHash=0xf7a5643748e04f842aeb369715a9fbc982caee565457126bb533d58b40c4b604) when briding (instead of explorer link to the "send" tx)